### DISCUSSION TITLE
Animations

### DISCUSSION THREAD

**Discussion by HudsonSchumaker** [GRRLIB/GRRLIB#16] [Q&A] [answered]
Hi, 
There is any example of an sprite animation using GRRLIB?
Thanks in advance

**Comment by Crayon2000** [comment, marked as answer]
Look at the **basic_drawing** demo with the Link sprite moving:

- Code: https://github.com/GRRLIB/GRRLIB/tree/master/examples/basic_drawing
- Video: https://youtu.be/PlPdVqYZZ_g?t=7
- Artifacts download (if you dont't want to build it): https://github.com/GRRLIB/GRRLIB/actions/runs/878549811
